guides/controllers/params.md

```diff
@@ -25,14 +25,14 @@ To set a param use the following syntax
 params[:key] = "My String"
 ```
 
-Please note that while the params accepts a Symbol key to be passed, it will be converted to a String automatically. For example the following two statements are equivalent.
+Please note that while params accepts a Symbol key to be passed, it will be converted to a String automatically. For example the following two statements are equivalent.
 
 ```ruby
 params[:key] # => "My String"
 params["key"] # => "My String"
 ```
 
-To access the validated params hash. It will only contain entries that have been sucesssfully validated using the Validation API. If you do not use the Validation API this will return an empty Hash.
+To access the validated params hash. It will only contain entries that have been successfully validated using the Validation API. If you do not use the Validation API this will return an empty Hash.
```

Comment:
`To access the validated params hash.` is not a complete sentence.  Maybe replace it with `The params object will only contain entries that have been successfully validated using the Validation API.`

Reply:
updated